besu/src/main/scripts/unixStartScript.txt

```diff
@@ -182,19 +182,19 @@ APP_ARGS=`save "\$@"`
 # Collect all arguments for the java command, following the shell quoting and substitution rules
 eval set -- \$DEFAULT_JVM_OPTS \$JAVA_OPTS \$${optsEnvironmentVar} <% if ( appNameSystemProperty ) { %>"\"-D${appNameSystemProperty}=\$APP_BASE_NAME\"" <% } %>-classpath "\"\$CLASSPATH\"" <% if ( mainClassName.startsWith('--module ') ) { %>--module-path "\"\$MODULE_PATH\"" <% } %>${mainClassName} "\$APP_ARGS"
 
-unset BESU_USING_JEMALLOC
 if [ "\$darwin" = "false" -a "\$msys" = "false" ]; then
-  # check if jemalloc is available
-  TEST_JEMALLOC=\$(LD_PRELOAD=libjemalloc.so sh -c true 2>&1)
-
-  # if jemalloc is available the output is empty, otherwise the output has an error line
-  if [ -z "\$TEST_JEMALLOC" ]; then
-    export LD_PRELOAD=libjemalloc.so
-    export BESU_USING_JEMALLOC=true
-  else
-    # jemalloc not available, as fallback limit malloc to 2 arenas
-    export MALLOC_ARENA_MAX=2
-  fi
+      if [ "\$BESU_USING_JEMALLOC" != "FALSE" -a "\$BESU_USING_JEMALLOC" != "false" ]; then
```

Comment:
@fab-10 is the intent correct? i.e. we only want to attempt to test and update LD_PRELOAD `libjemalloc` from startup script if and only if `BESU_USING_JEMALLOC` is not set to FALSE? 

My understanding was that we **always** attempt to load libjemalloc if we are able to test/load it via startup script and then set the variable BESU_USING_JEMALLOC so that we can log the JEMALLOC library version in our code at start up. (Because we never explicitly load jemalloc, it's just there available for JVM to use to allocate memory etc.)

I thought that `BESU_USING_JEMALLOC` is a misleading variable, it should be renamed to something like: `LOG_JEMALLOC_VERSION_AT_STARTUP` to clear the intent.

Anyways, I have removed my change request from the the PR, it is ready to merge. I'll leave it for you to proceed with the merge please.

Reply:
yes this is now the intent, disable `jemalloc` if `BESU_USING_JEMALLOC=false`, so we keep the original auto-detect mechanism if the env var is not set.

@amsmota I will set auto merge on, please merge again with main

Reply:
Hi guys, thanks for that. 

@usmansaleem, the objective of this PR was based on an issue we had when deploying in Prod some Besu nodes, deployment that succeed in the lower environments but not in Prod, so we had to remove Jemalloc and redeploy. I thougth then that if we had a env var to control the load/no-load it we could simply have set that env var to false and the deploy could go ahead without delays. And it ocurred to me that this could be used also for testing the performance gains of using Jemalloc, allowing for quickly testing the same app with our without it very quickly. Please note that if the env var does not exist at all Besu will behave exactly as it is now, it will output the current message if jemalloc does not exists ("jemalloc library not found, memory usage may be reduced by installing it") or goes ahead and prints the jemalloc version on the initial output of Besu.

@fab-10, so IIUC that means that I simply have to close the PR the merge will go ahead, so I'll do it now.

Again I would like to thank you guys for helping me understand the way jemalloc was loaded and reported on the logs, and changing my initial draft according to that.

Cheers guys.